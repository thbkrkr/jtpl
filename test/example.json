{
    "SSLPath":"/etc/ssl/my.domain",
    "Servers": [
      {
        "Name": "api1",
        "Url": "api1.my.domain:42"
      },
      {
        "Name": "api2",
        "Url": "api2.my.domain:42"
      }
    ]
}