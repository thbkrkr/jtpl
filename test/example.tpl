# A test template

ssl {{.SSLPath}}

{{range $server := .Servers -}}
server {{$server.Name}} {{$server.Url}}
{{end}}