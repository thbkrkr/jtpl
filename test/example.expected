# A test template

ssl /etc/ssl/my.domain

server api1 api1.my.domain:42
server api2 api2.my.domain:42